### ISSUE TITLE
Can't find Python 3.7 linker.

### ISSUE THREAD

**Issue by moorepants** [conda-forge/python-feedstock#298]
Issue:

Yesterday my CI service based on conda-forge's Python 3.7 was working fine:

https://travis-ci.org/pydy/pydy/jobs/629225276?utm_medium=notification&utm_source=github_status

It was installing Python 3.7.3.

Today, it is breaking with this error:

```
Exception: Compilation failed (return status=1): /usr/bin/ld: cannot find -lpython3.7. collect2: error: ld returned 1 exit status. 
```

See: https://travis-ci.org/pydy/pydy/jobs/629442673?utm_medium=notification&utm_source=github_status

It is downloading: `python                    3.7.6                h357f687_0    conda-forge`.

I'm not sure why it was linking properly with 3.7.3 yesterday and now fails with 3.7.6.

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list
$ conda list

# packages in environment at /home/travis/miniconda/envs/test-env:

#

# Name                    Version                   Build  Channel

_libgcc_mutex             0.1                        main    conda-forge

alabaster                 0.7.12                     py_0    conda-forge

attrs                     19.3.0                     py_0    conda-forge

babel                     2.7.0                      py_0    conda-forge

backcall                  0.1.0                      py_0    conda-forge

binutils_impl_linux-64    2.33.1               he1b5a44_7    conda-forge

binutils_linux-64         2.33.1              h9595d00_15    conda-forge

bleach                    3.1.0                      py_0    conda-forge

ca-certificates           2019.11.28           hecc5488_0    conda-forge

certifi                   2019.11.28               py37_0    conda-forge

cffi                      1.13.2           py37h8022711_0    conda-forge

chardet                   3.0.4                 py37_1003    conda-forge

coverage                  5.0.1            py37h516909a_0    conda-forge

cryptography              2.8              py37h72c5cf5_1    conda-forge

cycler                    0.10.0                     py_2    conda-forge

cython                    0.29.14          py37he1b5a44_0    conda-forge

dbus                      1.13.6               he372182_0    conda-forge

decorator                 4.4.1                      py_0    conda-forge

defusedxml                0.6.0                      py_0    conda-forge

docutils                  0.15.2                   py37_0    conda-forge

entrypoints               0.3                   py37_1000    conda-forge

expat                     2.2.5             he1b5a44_1004    conda-forge

fastcache                 1.1.0            py37h516909a_0    conda-forge

flake8                    3.7.9                    py37_0    conda-forge

fontconfig                2.13.1            h86ecdb6_1001    conda-forge

freetype                  2.10.0               he983fc9_1    conda-forge

gcc_impl_linux-64         7.3.0                habb00fd_2    conda-forge

gcc_linux-64              7.3.0               h553295d_15    conda-forge

gettext                   0.19.8.1          hc5be6a0_1002    conda-forge

glib                      2.58.3          py37h6f030ca_1002    conda-forge

gmp                       6.1.2             hf484d3e_1000    conda-forge

gmpy2                     2.1.0b1          py37h04dde30_0    conda-forge

gst-plugins-base          1.14.5               h0935bb2_0    conda-forge

gstreamer                 1.14.5               h36ae1b5_0    conda-forge

gxx_impl_linux-64         7.3.0                hdf63c60_2    conda-forge

gxx_linux-64              7.3.0               h553295d_15    conda-forge

icu                       64.2                 he1b5a44_1    conda-forge

idna                      2.8                   py37_1000    conda-forge

imagesize                 1.1.0                      py_0    conda-forge

importlib_metadata        1.3.0                    py37_0    conda-forge

ipydatawidgets            4.0.1                      py_0    conda-forge

ipykernel                 5.1.3            py37h5ca1d4c_0    conda-forge

ipyscales                 0.3.0                      py_0    conda-forge

ipython                   7.10.1           py37h5ca1d4c_0    conda-forge

ipython_genutils          0.2.0                      py_1    conda-forge

ipywidgets                7.5.1                      py_0    conda-forge

jedi                      0.15.2                   py37_0    conda-forge

jinja2                    2.10.3                     py_0    conda-forge

jpeg                      9c                h14c3975_1001    conda-forge

jsonschema                3.2.0                    py37_0    conda-forge

jupyter_client            5.3.3                    py37_1    conda-forge

jupyter_core              4.6.1                    py37_0    conda-forge

jupyter_sphinx            0.2.3                    py37_0    conda-forge

kiwisolver                1.1.0            py37hc9558a2_0    conda-forge

ld_impl_linux-64          2.33.1               h53a641e_7    conda-forge

libblas                   3.8.0               14_openblas    conda-forge

libcblas                  3.8.0               14_openblas    conda-forge

libclang                  9.0.0           default_hde54327_4    conda-forge

libffi                    3.2.1             he1b5a44_1006    conda-forge

libgcc-ng                 9.2.0                hdf63c60_0    conda-forge

libgfortran-ng            7.3.0                hdf63c60_2    conda-forge

libgpuarray               0.7.6             h14c3975_1003    conda-forge

libiconv                  1.15              h516909a_1005    conda-forge

liblapack                 3.8.0               14_openblas    conda-forge

libllvm9                  9.0.0                hc9558a2_3    conda-forge

libopenblas               0.3.7                h5ec1e0e_6    conda-forge

libpng                    1.6.37               hed695b0_0    conda-forge

libsodium                 1.0.17               h516909a_0    conda-forge

libstdcxx-ng              9.2.0                hdf63c60_0    conda-forge

libuuid                   2.32.1            h14c3975_1000    conda-forge

libxcb                    1.13              h14c3975_1002    conda-forge

libxkbcommon              0.9.1                hebb1f50_0    conda-forge

libxml2                   2.9.10               hee79883_0    conda-forge

mako                      1.1.0                      py_0    conda-forge

markupsafe                1.1.1            py37h516909a_0    conda-forge

matplotlib                3.1.2                    py37_1    conda-forge

matplotlib-base           3.1.2            py37h250f245_1    conda-forge

mccabe                    0.6.1                      py_1    conda-forge

mistune                   0.8.4           py37h516909a_1000    conda-forge

more-itertools            8.0.2                      py_0    conda-forge

mpc                       1.1.0             h04dde30_1006    conda-forge

mpfr                      4.0.2                he80fd80_0    conda-forge

mpmath                    1.1.0                      py_0    conda-forge

nbconvert                 5.6.1                    py37_0    conda-forge

nbformat                  4.4.0                      py_1    conda-forge

ncurses                   6.1               hf484d3e_1002    conda-forge

nose                      1.3.7                 py37_1003    conda-forge

notebook                  6.0.1                    py37_0    conda-forge

nspr                      4.24                 he1b5a44_0    conda-forge

nss                       3.47                 he751ad9_0    conda-forge

numpy                     1.17.3           py37h95a1406_0    conda-forge

numpydoc                  0.9.2                      py_0    conda-forge

openssl                   1.1.1d               h516909a_0    conda-forge

packaging                 19.2                       py_0    conda-forge

pandoc                    2.9.1                         0    conda-forge

pandocfilters             1.4.2                      py_1    conda-forge

parso                     0.5.2                      py_0    conda-forge

pcre                      8.43                 he1b5a44_0    conda-forge

pexpect                   4.7.0                    py37_0    conda-forge

pickleshare               0.7.5                 py37_1000    conda-forge

pip                       19.3.1                   py37_0    conda-forge

prometheus_client         0.7.1                      py_0    conda-forge

prompt_toolkit            3.0.2                      py_0    conda-forge

pthread-stubs             0.4               h14c3975_1001    conda-forge

ptyprocess                0.6.0                   py_1001    conda-forge

pycodestyle               2.5.0                      py_0    conda-forge

pycparser                 2.19                     py37_1    conda-forge

pyflakes                  2.1.1                      py_0    conda-forge

pygments                  2.5.2                      py_0    conda-forge

pygpu                     0.7.6           py37hc1659b7_1000    conda-forge

pyopenssl                 19.1.0                   py37_0    conda-forge

pyparsing                 2.4.6                      py_0    conda-forge

pyqt                      5.12.3           py37hcca6a23_1    conda-forge

pyqt5-sip                 4.19.18                  pypi_0    pypi

pyqtwebengine             5.12.1                   pypi_0    pypi

pyrsistent                0.15.6           py37h516909a_0    conda-forge

pysocks                   1.7.1                    py37_0    conda-forge

python                    3.7.6                h357f687_0    conda-forge

python-dateutil           2.8.1                      py_0    conda-forge

pythreejs                 2.1.1                      py_0    conda-forge

pytz                      2019.3                     py_0    conda-forge

pyzmq                     18.1.1           py37h1768529_0    conda-forge

qt                        5.12.5               hd8c4c69_1    conda-forge

readline                  8.0                  hf8c457e_0    conda-forge

requests                  2.22.0                   py37_1    conda-forge

scipy                     1.4.1            py37h921218d_0    conda-forge

send2trash                1.5.0                      py_0    conda-forge

setuptools                42.0.2                   py37_0    conda-forge

six                       1.13.0                   py37_0    conda-forge

snowballstemmer           2.0.0                      py_0    conda-forge

sphinx                    2.3.1                      py_0    conda-forge

sphinxcontrib-applehelp   1.0.1                      py_0    conda-forge

sphinxcontrib-devhelp     1.0.1                      py_0    conda-forge

sphinxcontrib-htmlhelp    1.0.2                      py_0    conda-forge

sphinxcontrib-jsmath      1.0.1                      py_0    conda-forge

sphinxcontrib-qthelp      1.0.2                      py_0    conda-forge

sphinxcontrib-serializinghtml 1.1.3                      py_0    conda-forge

sqlite                    3.30.1               hcee41ef_0    conda-forge

sympy                     1.5                      py37_0    conda-forge

terminado                 0.8.3                    py37_0    conda-forge

testpath                  0.4.4                      py_0    conda-forge

theano                    1.0.4           py37hf484d3e_1000    conda-forge

tk                        8.6.10               hed695b0_0    conda-forge

tornado                   6.0.3            py37h516909a_0    conda-forge

traitlets                 4.3.3                    py37_0    conda-forge

traittypes                0.2.1                      py_1    conda-forge

urllib3                   1.25.7                   py37_0    conda-forge

wcwidth                   0.1.7                      py_1    conda-forge

webencodings              0.5.1                      py_1    conda-forge

wheel                     0.33.6                   py37_0    conda-forge

widgetsnbextension        3.5.1                    py37_0    conda-forge

xorg-libxau               1.0.9                h14c3975_0    conda-forge

xorg-libxdmcp             1.1.3                h516909a_0    conda-forge

xz                        5.2.4             h14c3975_1001    conda-forge

zeromq                    4.3.2                he1b5a44_2    conda-forge

zipp                      0.6.0                      py_0    conda-forge

zlib                      1.2.11            h516909a_1006    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info
$ conda info

     active environment : test-env

    active env location : /home/travis/miniconda/envs/test-env

            shell level : 1

       user config file : /home/travis/.condarc

 populated config files : /home/travis/.condarc

          conda version : 4.6.14

    conda-build version : not installed

         python version : 2.7.14.final.0

       base environment : /home/travis/miniconda  (writable)

           channel URLs : https://conda.anaconda.org/conda-forge/linux-64

                          https://conda.anaconda.org/conda-forge/noarch

                          https://repo.anaconda.com/pkgs/main/linux-64

                          https://repo.anaconda.com/pkgs/main/noarch

                          https://repo.anaconda.com/pkgs/free/linux-64

                          https://repo.anaconda.com/pkgs/free/noarch

                          https://repo.anaconda.com/pkgs/r/linux-64

                          https://repo.anaconda.com/pkgs/r/noarch

          package cache : /home/travis/miniconda/pkgs

                          /home/travis/.conda/pkgs

       envs directories : /home/travis/miniconda/envs

                          /home/travis/.conda/envs

               platform : linux-64

             user-agent : conda/4.6.14 requests/2.22.0 CPython/2.7.14 Linux/4.15.0-1028-gcp ubuntu/16.04.5 glibc/2.23

                UID:GID : 2000:2000

             netrc file : None

           offline mode : False


```
</details>

**Comment by mingwandroid** [comment]
This should be fixed now. Cheers.

**Comment by isuruf** [comment]
Fixed in #299

**Comment by moorepants** [comment]
Thanks, it is working now!
